manifests/m/Microsoft/DataTools/IntegrationServices/3.14/Microsoft.DataTools.IntegrationServices.yaml

```diff
@@ -0,0 +1,22 @@
+PackageName: SQL Server Integration Services Projects
+Publisher: Microsoft Corporation
+PackageIdentifier: Microsoft.DataTools.IntegrationServices
+PackageVersion: 3.14
+License: Copyright (c) Microsoft Corporation. All rights reserved.
+ShortDescription: This project may be used for building high performance data integration and workflow solutions, including extraction, transformation, and loading (ETL) operations for data warehousing.
+PackageUrl: https://marketplace.visualstudio.com/items?itemName=SSIS.SqlServerIntegrationServicesProjects
+Tags: 
+- "dtproj"
+- "Visual Studio"
+- "SSIS"
```

Comment:
```suggestion
- dtproj
- visual-studio
- ssis
```

Reply:
@ItzLevvie Did your virtual machine have Visual Studio 2019 installed?  This is an extension for VS that is packaged as an EXE.

Reply:
cc @philipwolfe: No, all manifests are validated on a clean installation of Windows with no dependencies installed.

This package has an install-time dependency, so it looks like you'll have to wait for dependency support to be added which is being tracked at https://github.com/microsoft/winget-cli/issues/163.

So please add your upvote there.